src/_P002_ADC.ino

```diff
@@ -18,10 +18,76 @@
   # define P002_MAX_ADC_VALUE    1023
 #endif // ifdef ESP8266
 
-uint32_t Plugin_002_OversamplingValue  = 0;
-uint16_t Plugin_002_OversamplingCount  = 0;
-uint16_t Plugin_002_OversamplingMinVal = P002_MAX_ADC_VALUE;
-uint16_t Plugin_002_OversamplingMaxVal = 0;
+
+#define P002_OVERSAMPLING        PCONFIG(0)
+#define P002_CALIBRATION_ENABLED PCONFIG(3)
```

Comment:
also bitfield?

Reply:
will be incompatible with old config... never mind

Reply:
Yep it is indeed a matter of existing settings.